scripts/schmidt_comparison.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(0,'../chamber/models')
+
+from CoolProp import HumidAirProp as hap
+import pandas as pd
+
+import props
+
+
+results = {'Pressure (Pa)': [], 'Temperature (K)': [],
+					'Ralative Humidity': [], 'Schmidt Mills': [],
+					'Schmidt Marrero': []}
+
+
+# Gets relevant values and calculates Schmidt number
+def get_schmidt(p, t, rh):
```

Comment:
This function should live outside the script. I am inclined to place it in `models.props` however, this is not really a property, but rather a nondimensional parameter. I think we should start a new module (.py file) called `params` that will be responsible for calculating all the nondimensional parameters; e.g. Gr, Nu, Pr, Sc, Sh, etc.

Here is what I suggest:
  1. Create a file in chamber/models, name it `params.py` and place the `get_schmidt` function there. Then when you `import chamber.models as mod` you can call `mod.get_schmidt()`.
  2. Also, let's be explicit with the `Mills` or `Marrero` as an input, called `ref`. Then use an `if` statement to determine what logic to use. There are examples of this in `props`. Let me know if you have a question.